sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -37,10 +37,10 @@ protected void beforeTest() {
 
         if (interceptorManager.isPlaybackMode()) {
             client = clientSetup(credentials -> new ConfigurationClientBuilder()
-                    .credential(credentials)
-                    .httpClient(interceptorManager.getPlaybackClient())
-                    .httpLogDetailLevel(HttpLogDetailLevel.BODY_AND_HEADERS)
-                    .buildAsyncClient());
+                .credential(credentials)
```

Comment:
Doe this fail checkstyle? I remember having to revert some auto-formatting intellij did. Also, we added suppression for tests?

Reply:
No suppression for tests. I believe tests should also meet the 120 lines, 
The only suppression I consider was the samples.

Reply:
We definitely need to suppress this on storage tests, they use data grids which I don't think can handle new lines.